docs/markdown/CMake-module.md

```diff
@@ -273,15 +275,15 @@ conf = configuration_data()
 conf.set_quoted('VAR', 'variable value')
 
 cmake.configure_package_config_file(
-    name: 'myProject',
-    input: 'myProject.cmake.in',
-    configuration: conf
+  name: 'myProject',
+  input: 'myProject.cmake.in',
+  configuration: conf
```

Comment:
All the other changes in this file are now:
- the doc updates
- correcting a link
- fixing a syntax highlight issue
- using the correct capitalization for CMake

But this here is a style change  for differing tab length conventions (and changing from the style I personally like to one I don't like). Can we skip this diff hunk?

Reply:
Oke, I only kept the link update because it pointed to CMake 3.11, where ARCH_INDEPENDENT isn't defined